static/app/components/modals/addDashboardIssueWidgetModal.tsx

```diff
@@ -62,7 +63,7 @@ type State = {
 
 const newQuery = {
   name: '',
-  fields: [] as string[],
+  fields: ['issue', 'title', 'assignee'] as string[],
```

Comment:
nit: can we order it `['issue', 'assignee', 'title']` so that assignee does get cut off because of a long title

Reply:
👍  done!